competition-simulator/programming/time.md

```diff
@@ -1,62 +1,23 @@
 ---
 layout: page
-title: Simulated Time
+title: Robot Time
 ---
 
-# Simulated Time
+# Important methods
 
 In the simulated environment, time advances only at the pace that the simulator
 is run and in cooperation with the controller code (such as the code controlling
-your robot). As a result, using `time.time` to know how long your robot has been
-running for or `time.sleep` to wait for some duration may be unreliable.
+your robot). As a result, using the `time` method to know how long your robot has been
+running for or `sleep` method to wait for some duration is the most reliable way of 
+implementing precise movements in your robot
```

Comment:
I'm not sure that this is something that we want to say here. Partly because we don't want to encourage dead-reckoning (i.e: purely time based movements), but also because it may not be true. For example if the robot isn't using its brakes then it is likely that the robot will have travelled a longer than expected amount. Alternatively if competitors use the vision system and/or we offer competitors another locations method (perhaps such as an IMU or wheel position), then using time for precise movements will no longer be the best method.

Instead I suggest that we keep something along the lines of the previous paragraph here, though perhaps strengthen what it says so that it is clear that using the `time` module shouldn't be used.

Reply:
b126b3c